### PR TITLE
31 Add possibility to remove docker

### LINKED ISSUE #31: Unable to uninstall docker
I usually use your `docker` role to install/uninstall docker in systems.

Since I have to manage some raspberry systems, I also use your `docker_arm` role.

Today, I want to remove docker from one raspberry. So I configured the variable `docker_package_state` to "absent", like I do this on all other system. Now I must learn, that the `docker_arm` role doesn't support this.

It would be great, if the role also support removing docker on the same way the `docker` role does.

### PR DESCRIPTION
**Pull request by FrankGiesecke**
Add the possibility to remove docker using the same variable `docker_package_state` as the docker role does.

Fixes #31

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -22,32 +22,35 @@
...
+    state: "{{ docker_package_state }}"
     force: true
     install_recommends: "{{ docker_install_recommends }}"
 
-# Shim for Docker 18.09.0 on Debian Stretch.
-- import_tasks: docker-1809-shim.yml
-  when:
-    - ansible_service_mgr == 'systemd'
-    - "'18.09.0' in docker_version"
+- name: process docker
```
Comment (geerlingguy) [inline tasks/main.yml:29]: This seems to be a separate / unrelated change?

File: defaults/main.yml [hunk 2]
```diff
@@ -17,3 +17,5 @@ docker_users: []
 
 # The executable to use for Pip.
 docker_pip_executable: pip3
+
+docker_package_state: present
```
Comment (geerlingguy) [inline defaults/main.yml:21]: This new variable also needs to be documented in the README file.

**Review by geerlingguy — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (FrankGiesecke) [reply tasks/main.yml:29]: Sorry for this.

I've moved some tasks (`docker-1809-shim.yml`, `Ensure Docker is started` and `Ensure docker users are added to the docker group`) into a block to avoid them when uninstalling docker. 

IMHO: these tasks shouldn't be executed if there is no docker.

**Review by geerlingguy — APPROVED** [review]

**Comment by FrankGiesecke** [comment]
@geerlingguy When do you plan to merge these changes?

**Comment by geerlingguy** [comment]
This role has been deprecated in favor of my generic docker role.
